citi-onboarding/DADe-onepage
# Feature/navbar and footer mobile

## What I did

- Update Navbar look and colors to look like the new mockup
- Mobile  navbar
- Mobile footer
- obs: Navbar burger menu mockup is old, I already removed the 'O que é o D.A?' since it no longer exists in the new mockup navbar.

| Mockup | Real |
|--------|------|
|![nav_mock](https://user-images.githubusercontent.com/61971582/122941451-c9685d00-d34b-11eb-80be-4a6a27ee8673.png) | ![nav_real](https://user-images.githubusercontent.com/61971582/122941548-dd13c380-d34b-11eb-9d31-a9d0fbef7617.png) |

| Mockup | Real |
|--------|------|
|![nav_desktop_mock](https://user-images.githubusercontent.com/61971582/122941478-ce2d1100-d34b-11eb-808a-794ff91deac6.png)|![nav_desktop_real](https://user-images.githubusercontent.com/61971582/122941556-dedd8700-d34b-11eb-80f0-38fd892d5012.png)|

| Mockup | Real |
|--------|------|
|![mock_footer](https://user-images.githubusercontent.com/61971582/122941505-d38a5b80-d34b-11eb-9840-9b376f389cae.png)|![footer_real](https://user-images.githubusercontent.com/61971582/122941566-e1d87780-d34b-11eb-8a9f-b61df09e37cf.png)|

| Mockup | Real |
|--------|------|
|![menu_mock](https://user-images.githubusercontent.com/61971582/122941521-d71de280-d34b-11eb-9a12-e771ddc63568.png)|![Captura de Tela_select-area_20210622114209](https://user-images.githubusercontent.com/61971582/122945704-231e5680-d34f-11eb-8b79-aaa47abde4da.png)|

|

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feature/navbar-and-footer-mobile`
   1. `git pull origin feature/navbar-and-footer-mobile`
   1. `npm install`
   1. `cd server/`
   1.  `node index.js`
   1. Open a new terminal
   2. `cd client/`
   3. `npm start`

1. In your browser go to http://localhost:3000 (should open with `npm start`)

1. Check if both the navbar and the footer look just like the mockup in different screen sizes, and their mobile versions too. 
-obs: In this branch, a lot of sections are going overboard in small screens, so, you might need to use the horizontal scroll or tweak the zoom setting to actually get the full look at the navbar.

1. Check if you can click the mobile navbar menu and make it open, and if you can exit it by the "X" or by clicking in a section link (members and contact should already be working links).Check if when you scroll you can't see the page with the menu open (you should not be  able to see the page).

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description
